components/table/sortable-table.js

```diff
@@ -0,0 +1,239 @@
+function SortableTable(table) {
+
+    var table = table;
+    var status;
+  
+    var options = {};
+    options.statusMessage = 'Sort by %heading% (%direction%)';
```

Comment:
Ideally this text should come from a data attribute so that this can support other languages